src/iop/zonesystem.c

```diff
@@ -203,7 +203,7 @@ static void process_common_cleanup(struct dt_iop_module_t *self, dt_dev_pixelpip
   const int ch = piece->colors;
   const int size = d->params.size;
 
-  if(piece->pipe->mask_display) dt_iop_alpha_copy(ivoid, ovoid, width, height);
+  if(piece->pipe->mask_display & DT_DEV_PIXELPIPE_DISPLAY_MASK) dt_iop_alpha_copy(ivoid, ovoid, width, height);
```

Comment:
Is there any particular reason why this line should be in the each and every `process()`/`process_sse2()`?
How about simply putting it here: https://github.com/darktable-org/darktable/blob/b0fa66a7b16f1502dfc38e35d96e66fdccc5a75d/src/develop/imageop.c#L189